cherrypy/test/test_params.py

```diff
@@ -0,0 +1,34 @@
+import cherrypy
+from cherrypy.test import helper
+
+
+class ParamsTest(helper.CPWebCase):
+    @staticmethod
+    def setup_server():
+        class Root:
+            @cherrypy.expose
+            @cherrypy.tools.params(exception=ValueError)
+            def resource(self, limit=None, sort=None):
+                return '{0}: {1}'.format(limit, type(limit).__name__)
+            # for testing on Py 2
```

Comment:
I like that this can be tested on Python 2, but I'd like also if on the appropriate Python versions if it used the native syntax (to validate that the native syntax does what you expect, but also to make the syntax visible). I'm not sure it's possible or won't be so ugly to implement that it violates the goals, so this is just something I'd like to see, but don't require.